recursive_art.py

```diff
@@ -20,9 +21,16 @@ def build_random_function(min_depth, max_depth):
         (See the assignment writ-eup for details on the representation of
         these functions)
     """
-    # TODO: implement this
-    pass
-
+    one = ['x','y']
+    function = ['x','y','cos_pi','sin_pi','prod','square','average','square_root']
+    if max_depth == 1:
+        return one[random.randint(0,1)]
```

Comment:
You could also use random.choice(one), that grabs a random element from the list and is (in my opinion, at least) a slightly cleaner way to handle it. That said, it's entirely your call!